discouraged

```diff
@@ -9182,8 +9181,8 @@
 "mdsymlem6" is used by "mdsymlem7".
 "mdsymlem7" is used by "mdsymlem8".
 "mdsymlem8" is used by "mdsymi".
-"measdivcstOLD" is used by "probfinmeasbOLD".
-"measdivcstOLD" is used by "probmeasb".
+"measdivcstALTV" is used by "probfinmeasbALTV".
+"measdivcstALTV" is used by "probmeasb".
```

Comment:
Neither an OLD nor an ALTV theorem should be used by a non-OLD/ALTV theorem. Can the proof of ~probmeasb be revised accordingly (using ~ measdivcst instead of measdivcstALTV)?

Reply:
> Neither an OLD nor an ALTV theorem should be used by a non-OLD/ALTV theorem.

I have some reservations about the truth of this statement in the general case. Notice how OLD proofs always require a `(New usage is discouraged.)` label, but that's not the case for ALTV proofs. Therefore, accoring to [conventions](https://us.metamath.org/mpeuni/conventions.html), some ALTV theorems can be picked up through a minimization process by non-OLD/ALTV theorems (if you don't want that, you would advocate for `minimize * /allow * /no ax-* /forbid *ALTV`, but at the moment that's not how default minimizations are indicated to be done).

So the question is whether the use of ~ measdivcst is preferred over measdivcstALTV in this particular case, but not because of a general rule.

Reply:
ALT does require `(New usage is discouraged.)`; I'm not sure what ALTV is supposed to indicate.

Reply:
> ALT does require `(New usage is discouraged.)`; I'm not sure what ALTV is supposed to indicate.

Yes ALT does, but ALTV does not, I read it from the conventions [here](https://us.metamath.org/mpeuni/conventions.html)

Reply:
ALT versions are supposed to be used when different proofs of the same theorem are provided, while ALTV means that the statement itself differs from the original. I think in the database some theorems who were supposed to be ALTV were labeled as ALT instead (example sbequ1 and sbequ1ALT) and this created confusion.

Reply:
When was that decided ? That seems rather peculiar to me. The database set.mm is full of "alternative versions", and there are no special labellings for them (in particular, no capitalization). Do you have examples ?

Reply:
> When was that decided ? That seems rather peculiar to me.

Everything I'm saying comes from the conventions, here is a copy-paste of them:

>Alternate (ALT) proofs. If a different proof is shorter or clearer but uses more or stronger axioms, we make that proof an "alternate" proof (marked with an ALT label suffix), even if this alternate proof was formalized first. We then make the proof that requires fewer axioms the main proof. Alternate proofs can also occur in other cases when an alternate proof gives some particular insight. Their comment should begin with "Alternate proof of ~ xxx " followed by a description of the specificity of that alternate proof. There can be multiple alternates. Alternate (*ALT) theorems should have "(Proof modification is discouraged.) (New usage is discouraged.)" in their comment and should follow the main statement, so that people reading the text in order will see the main statement first. The alternate and main statement comments should use hyperlinks to refer to each other.

>Alternative (ALTV) versions. If a theorem or definition is an alternate/variant of an existing theorem or definition, its label should have the same name with suffix ALTV. Such alternates are often temporary only, until it is decided which alternative should be used in the future. Alternate (*ALTV) theorems or definitions are usually contained in mathboxes. Their comments need not to contain "(Proof modification is discouraged.) (New usage is discouraged.)". Their comment should begin with "Alternate version of ~ xxx " followed by a description of the specificity of the difference from the main variant. Alternate statements should generally follow the main statement.

>Old (OLD) versions or proofs. If a proof, definition, axiom, or theorem is going to be removed, we often stage that change by first renaming its label with an OLD suffix (to make it clear that it is going to be removed). Old (*OLD) statements should have "(Proof modification is discouraged.) (New usage is discouraged.)" and "Obsolete version of ~ xxx as of dd-Mmm-yyyy." (not enclosed in parentheses) in the comment. An old statement should follow the main statement, so that people reading the text in order will see the main statement first. This typically happens when a shorter proof to an existing theorem is found: the existing theorem is kept as an *OLD statement for one year. When a proof is shortened automatically (using the Metamath program "MM-PA> MINIMIZE__WITH *" command), then it is not necessary to keep the old proof, nor to add credit for the shortening.

It clearly states that ALT suffixes are for alternative proofs, while ALTV suffixes are for variants of the original theorem.

> The database set.mm is full of "alternative versions", and there are no special labellings for them (in particular, no capitalization). Do you have examples ?

I'm not sure I understand, you mean examples of already existing ALTV theorems? If yes than there are several of them, such as `uniqs` in main and `uniqsALTV` is in Peter Mazsa's mathbox (all ALTV versions are in mathboxes as far as I know).

Reply:
Ok. I didn't remember that. Since *ALTV is a mathbox-only suffix, this should appear in the conventions. Also, it seems that this MR has *ALTV statements in the main part of set.mm, which should not be the case.

Reply:
> Also, it seems that this MR has *ALTV statements in the main part of set.mm, which should not be the case.

Not true, the title of this PR clearly states that I only touched @tirix mathbox, and that's what I did.

Reply:
If it's mathbox only, then fair enough.

Reply:
> > Neither an OLD nor an ALTV theorem should be used by a non-OLD/ALTV theorem.
> 
> I have some reservations about the truth of this statement in the general case. Notice how OLD proofs always require a `(New usage is discouraged.)` label, but that's not the case for ALTV proofs. Therefore, accoring to [conventions](https://us.metamath.org/mpeuni/conventions.html), some ALTV theorems can be picked up through a minimization process by non-OLD/ALTV theorems (if you don't want that, you would advocate for `minimize * /allow * /no ax-* /forbid *ALTV`, but at the moment that's not how default minimizations are indicated to be done).
> 
> So the question is whether the use of ~ measdivcst is preferred over measdivcstALTV in this particular case, but not because of a general rule.

OK, it's not in the conventions, ut thee is no case in set.mm where an ALTV theorem is used in a proof of a non-OLD/ALTV theorem. So no change required now, but it must be replaced in the proof of ~probmeasb before ~probmeasb is moved to Main at the latest.